models/public/hrnetv2/accuracy-check.yaml

```diff
@@ -0,0 +1,77 @@
+models:
+
+
+  - name: hrnetv2_pytorch
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        model:   public/hrnetv2/hrnetv2.onnx
```

Comment:
if you want run onnx model, please use onnx_runtime as launcher.
Our current config formatting is not include running models via onnx importer, I think we add it later for all models if it will be necessary.
Please remove this section or change launcher on onnx_runtime